airflow-core/src/airflow/ui/src/pages/DagRuns.tsx

```diff
@@ -257,6 +276,15 @@ export const DagRuns = () => {
             placeHolder={translate("dags:filters.runIdPatternFilter")}
           />
         </Box>
+        <Box>
+          <SearchBar
+            defaultValue={filteredTriggeringUserNamePattern ?? ""}
+            hideAdvanced
+            hotkeyDisabled={true}
+            onChange={handleTriggeringUserNamePatternChange}
+            placeHolder={translate("dags:filters.triggeringUserNameFilter")}
+          />
```

Comment:
The placeholder is truncated a little bit. We should probably let the input expand to have the full text displayed.
<img width="961" height="270" alt="Screenshot 2025-07-25 at 16 22 46" src="https://github.com/user-attachments/assets/14b17473-39f3-4fe7-8f71-4d466217d68b" />